app/javascript/components/mockEmergencyContact.js

```diff
@@ -0,0 +1,30 @@
+let mockEmergencyContact = [
+    {
+        id: 1,
+        full_name: "Naruto Uzumaki",
+        relationship: "hokage",
+        phone_number: "1234567890",
+        user_id: 1
+    },
+    {
+        id: 2,
+        full_name: "Gaara",
+        relationship: "kazakage",
+        phone_number: "1234567890",
+        user_id: 1
+    },
+    {
+        id: 3,
+        full_name: "Kakashi Hatake",
+        relationship: "sensei",
+        phone_number: "1234567890",
+        user_id: 2
+    },
+    {
+        id: 4,
+        full_name: "Shikamaru Nara",
+        relationship: "proctor",
+        phone_number: "1234567890",
+        user_id: 2
+    },
```

Comment:
nitpick: It looks like trailing camas are not used in other places so this one could be removed.